lib/src/main/scala/spinal/lib/com/pmod/PMOD.scala

```diff
@@ -0,0 +1,23 @@
+package spinal.lib.com.pmod
+
+import spinal.core._
+
+abstract class PMODBundle extends Bundle {
+  val pin1 = Bits(1 bit)
+  val pin2 = Bits(1 bit)
+  val pin3 = Bits(1 bit)
+  val pin4 = Bits(1 bit)
+  val pin7 = Bits(1 bit)
+  val pin8 = Bits(1 bit)
+  val pin9 = Bits(1 bit)
+  val pin10 = Bits(1 bit)
+
+  def build()
```

Comment:
What is the build function ? is it to set the direction right ?
What's about using the IMasterSlave interface ?

Reply:
Make sense. Let me try to do this

Reply:
@Dolu1990 done! 
It looks cleaner with IMasterSlave